scripts/noaddons.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Elico Corp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import re
+
+ODOO_CONF = '/opt/odoo/etc/odoo.conf'
+DEFAULT_ADDONS_PATH = 'addons_path = /opt/odoo/sources/odoo/addons'
+
+
+def main():
+    contains_addons_path = False
+    with open(ODOO_CONF, 'r') as conf_file:
+        for line in conf_file:
+            if re.match(r'^addons_path*=*', line):
```

Comment:
I've seen this expressions too many times! Please put it in a constant as well (in addons.py)